packages/flutter_tools/gradle/module_plugin_loader.gradle

```diff
@@ -35,6 +38,19 @@ if (pluginsFile.exists()) {
 
 gradle.getGradle().projectsLoaded { g ->
     g.rootProject.beforeEvaluate { p ->
+        //let the plugin's output generated in flutterModule/build/host/${androidPlugin.name}.
+        p.subprojects { sp ->
+            if (pluginsNames.contains(sp.name.replace(":", ""))){
```

Comment:
does `name` contain `:`? in my local test it doesn't.

Reply:
also can this just be `object.plugins.android`? that is a list already.

Reply:
maybe someone else modified the `name` on `master` with another pr. I checked my local project , the `name` does not contained the `:` now.

Reply:
ok. it might have been a recent Gradle version bump. Let's keep the logic that strips `:` out in the variable specified below.